Misc/NEWS.d/next/IDLE/2019-06-04-23-27-33.bpo-37039.FN_fBf.rst

```diff
@@ -1 +1,4 @@
-Fix incorrect behavior of IDLE's "Zoom Height" command across platforms.
+Adjust "Zoom Height" zooms to individual screens by momemtarily maximizing
+the window the first time one is zoomed on a particular screen.  Zooms will
+not be correct if the resolution and scaling for a screen is subsequently
+changed.  While a window is maximized, "Zoom Height" has no effect.
```

Comment:
As far as I can tell, changing the resolution and/or scaling of a screen will simply cause Tk to observe a different screen resolution, after which the current implementation will again maximize a screen once to get the proper values for the new screen configuration.  So this is incorrect: "Zooms will not be correct if the resolution and scaling for a screen is subsequently changed."

The only cases where we could get things slightly wrong is when the dock/taskbar is moved or changed, or perhaps moving a window to a screen with identical effective resolution but with other differences, e.g. with/without a dock/taskbar.

Reply:
Of course.  The dict key will change and not be found.  I will simplify to "Zooms may not be correct if the screen is subsequently changed."  and then merge.